src/analysis/seminorm.lean

```diff
@@ -377,7 +377,7 @@ lemma mem_ball_zero : y ∈ ball p 0 r ↔ p y < r := by rw [mem_ball, sub_zero]
 
 lemma ball_zero_eq : ball p 0 r = { y : E | p y < r } := set.ext $ λ x, p.mem_ball_zero
 
-lemma zero_ball_eq_univ (hr : 0 < r) : ball (0 : seminorm 𝕜 E) x r = set.univ :=
```

Comment:
`zero_ball` was also a fine name, but if you use that name you'll need to rename `ball_sup` to `sup_ball` etc.